test/e2e/elasticsearch_token_propagation_test.go

```diff
@@ -334,6 +334,17 @@ func testAccountToken() string {
 }
 
 func jaegerInstance() *v1.Jaeger {
+	var esResourcesRequest corev1.ResourceList
+	if sepcifyRequestMemory {
+		esResourcesRequest = corev1.ResourceList{
+			corev1.ResourceMemory: resource.MustParse(requestMemory),
+			corev1.ResourceCPU:    resource.MustParse(requestCPU),
+		}
+	} else {
+		esResourcesRequest = corev1.ResourceList{
+			corev1.ResourceMemory: resource.MustParse(requestMemory),
```

Comment:
Why are you specifying a memory request even when "specifyMemoryRequest" is false? And same question as the other PR: why do you have a CPU on the previous code branch, but not here?